utils/handlers/graphql.go

```diff
@@ -18,19 +19,32 @@ func HandleGraphQLRequest(graphql *graphql.Module) http.HandlerFunc {
 		_, cancel := context.WithTimeout(r.Context(), 5*time.Second)
 		defer cancel()
 
-		// Get the path parameters
-		_ = getRequestMetaData(r)
+		vars := mux.Vars(r)
+		projectID := vars["project"]
+		pid := graphql.GetProjectID()
+		if projectID == pid {
```

Comment:
Instead of checking for equality, check for inequality and return if its unequal